display/bookmarks.go

```diff
@@ -41,8 +41,10 @@ func bkmkInit() {
 		form.SetLabelColor(config.GetColor("bkmk_modal_label"))
 		form.SetFieldBackgroundColor(config.GetColor("bkmk_modal_field_bg"))
 		form.SetFieldTextColor(config.GetColor("bkmk_modal_field_text"))
+		form.SetFieldBackgroundColorFocused(config.GetColor("bkmk_modal_field_text"))
+		form.SetFieldTextColorFocused(config.GetTextColor("bkmk_modal_field_bg", "bkmk_modal_field_text"))
 		form.SetButtonBackgroundColorFocused(config.GetColor("btn_text"))
-		form.SetButtonTextColorFocused(config.GetColor("btn_bg"))
+		form.SetButtonTextColorFocused(config.GetTextColor("btn_bg", "btn_text"))
```

Comment:
Isn't this function call, and all the others in this PR, incorrect? The function signature for `GetTextColor` is `GetTextColor(key, bg string) tcell.Color`, which would indicate having `"btn_bg"` second and `"btn_text"` first. Or am I misunderstanding?

Reply:
for focused UI elements, the text color and bg color swap. When focused, the button has text color of whatever `"btn_bg"` is set to, and a bg of whatever color `"btn_text"` is set to.

Because `"default"` is not a valid text color in the config, this function is always called with a key ending with `bg` as its key, and a `bg` ending with text. 

I tried to be as concise as possible with the naming. Maybe `SwapColor(new_text_key, new_bg_key string)` or something similar would be less confusing.

Reply:
No, I think it's good the way it is then, I just misunderstood.